source/reference/kubectl-minio-plugin.rst

```diff
@@ -89,6 +89,71 @@ Subcommands
 - :mc:`~kubectl minio delete`
 - :mc:`~kubectl minio version`
 
+Environment Variables
+---------------------
+
+The :ref:`MinIO Operator <minio-operator-installation>` uses the following environment variables during startup to set configuration settings.
+
+
+.. envvar:: MINIO_OPERATOR_CERTIFICATES_VERSION
+
+   Specifies the certificate API version to use.
+
+   Valid values are ``v1`` or ``v1beta1``.
+
+   When not specified, the default is the API the Kubernetes provides.
+
+.. envvar:: MINIO_OPERATOR_RUNTIME
+
+   Specify the type of runtime to use.
+
+   Valid values are ``EKS``, ``Rancher``, or ``OpenShift``.
+   Leave blank if none of the value options apply.
+
+   When set as ``EKS``, the :envvar:`MINIO_OPERATOR_CSR_SIGNER_NAME` must be ``beta.eks.amazonaws.com/app-serving``.
+
+.. envvar:: MINIO_OPERATOR_CSR_SIGNER_NAME
+
+   Override the default signer for certificate signing requests (CSRs).
+
+   When not specified, the default value is ``kubernetes.io/kubelet-serving``.
+
+.. envvar:: OPERATOR_CERT_PASSWD
+   
+   *Optional*
+
+   The password Operator should use to use to decrypt the private key in the TLS certificate for Operator.
```

Comment:
I suspect this was intended to be `The password Operator should use to decrypt...`. Perhaps an alternative is "The password Operator should use to decrypt the private key in the Operator TLS certificate" (although I'm not entirely clear the context).

Reply:
Sometimes my fingers and mind don't quite stay in sync and one wants to do what the other already did.